xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -197,6 +197,31 @@ bool CDRMUtils::SupportsProperty(struct drm_object *object, const char *name)
   return false;
 }
 
+bool CDRMUtils::SupportsPropertyAndValue(struct drm_object* object,
+                                         const char* name,
+                                         uint64_t value /* = 0 */)
+{
+  for (uint32_t i = 0; i < object->props->count_props; i++)
+  {
+    if (strcmp(object->props_info[i]->name, name) != 0)
+      continue;
+
+    if (drm_property_type_is(object->props_info[i], DRM_MODE_PROP_ENUM) != 0)
+    {
+      for (int j = 0; j < object->props_info[i]->count_enums; j++)
+      {
+        if (object->props_info[i]->enums[j].value == value)
+          return true;
+      }
+    }
```

Comment:
The logging should be placed here after this `if` and for a small optimization it should be followed by a `break;` as we have found the property and do not expect there to be a second one.

Reply:
updated